format_spec/filters/webp.md

```diff
@@ -0,0 +1,60 @@
+---
+title: WEBP Filter
+---
+
+The WebP filter compresses image data using [libwebp](https://developers.google.com/speed/webp/docs/api#headers_and_libraries). The current input formats supported are `RGB`, `RGBA`, `BGR`, and `BGRA`. The colorspace format can be configured for this filter by setting the `TILEDB_WEBP_INPUT_FORMAT` filter option. Tile-based compression is determined by dimension extents. Extents equal to dimension bounds will compress the image in one pass.
```

Comment:
Thanks, this image will help to explain with a visual example. I'm working on updating this and poking at CI a bit this morning so this will be in shortly.